src/DIRAC/WorkloadManagementSystem/JobWrapper/Watchdog.py

```diff
@@ -847,9 +827,9 @@ def __getUsageSummary(self):
         if "MemoryUsed" in self.parameters:
             memory = self.parameters["MemoryUsed"]
             if memory:
-                summary["MemoryUsed(kb)"] = abs(float(memory[-1]) - float(self.initialValues["MemoryUsed"]))
+                summary["MemoryUsed(mb)"] = abs(float(memory[-1]) - float(self.initialValues["MemoryUsed"]))
```

Comment:
I think that computing the MemoryUsed as the difference between the last measured value of RSS and the initial value is fine in most of the cases. However it seems to me that it does not take into account eventual peaks of memory usage during the program execution. So I'm wondering if it wouldn't make sense to add another parameter "PeakMemoryUsed" or "MaxMemoryUsed". 
Anyway this could go in a separate PR.

Reply:
I think you can propose this in a different PR.

Reply:
ok fine for me. I will not do it immediately but I keep it in mind.